### DISCUSSION TITLE
Connection errored out with image prompt.

### DISCUSSION THREAD

**Discussion by Padraix** [lllyasviel/Fooocus#3098] [Q&A] [answered]
Hello. Normally, the program generates images, but when I use the image prompt option,   after clicking "generate image", I get a Connection errored out message. In the cmd window after running the program I have the following information:
D:\Programy\Fooocus AI>.\python_embeded\python.exe -s Fooocus\entry_with_update.py
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.4.1
[Cleanup] Attempting to delete content of temp dir C:\Users\Padraix\AppData\Local\Temp\fooocus
[Cleanup] Cleanup successful
You do not have [juggernautXL_v8Rundiffusion.safetensors] but you have [juggernautXL_version6Rundiffusion.safetensors].
Fooocus will use [juggernautXL_version6Rundiffusion.safetensors] to avoid downloading new models, but you are not using the latest models.
Use --always-download-new-model to avoid fallback and always get new models.
Total VRAM 6144 MB, total RAM 8190 MB
Set vram state to: NORMAL_VRAM
Always offload VRAM
Device: cuda:0 NVIDIA GeForce GTX 1660 : native
VAE dtype: torch.float32
Using pytorch cross attention
Refiner unloaded.
Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
IMPORTANT: You are using gradio version 3.41.2, however version 4.29.0 is available, please upgrade.
--------
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_g.logit_scale', 'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.logit_scale'}
Base model loaded: D:\Programy\Fooocus AI\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors
VAE loaded: None
Request to load LoRAs [('sd_xl_offset_example-lora_1.0.safetensors', 0.1)] for model [D:\Programy\Fooocus AI\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors].
Loaded LoRA [D:\Programy\Fooocus AI\Fooocus\models\loras\sd_xl_offset_example-lora_1.0.safetensors] for UNet [D:\Programy\Fooocus AI\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors] with 788 keys at weight 0.1.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cpu, use_fp16 = False.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
Started worker with PID 1976
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865
[Parameters] Adaptive CFG = 7
[Parameters] CLIP Skip = 2
[Parameters] Sharpness = 2
[Parameters] ControlNet Softness = 0.25
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 5876371819127033546
[Fooocus] Downloading control models ...
[Fooocus] Loading control models ...
extra clip vision: ['vision_model.embeddings.position_ids']
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 9
[Fooocus] Initializing ...
[Fooocus] Loading models ...

D:\Programy\Fooocus AI>pause
Press any key to continue . . .

My CMD: 

![error](https://github.com/lllyasviel/Fooocus/assets/155782314/95b5dec7-9b66-4151-8b7c-2aa3c8989f2e)

My Advanced System Settings:

![error 2](https://github.com/lllyasviel/Fooocus/assets/155782314/acc8645f-ae9e-4751-9ba6-8512a1a44857)

My Networking file: 

![error 3](https://github.com/lllyasviel/Fooocus/assets/155782314/548385c1-f904-4d75-a662-b4b2157d4b92)

I have: NVIDIA GeForce GTX 1660 GDDR5 6GB 
RAM: 8GB DDR3 
At Least 100 GB free space on both disk (C and D )

**Comment by mashb1t** [comment]
Please check the troubleshooting guide here: https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md
if this doesn't help please open an issue.

**Reply by Padraix** [reply, marked as answer]
OK, I did it. I added 40000 and 60000 in advanced system settings and now it works.
![00000-3594293049](https://github.com/lllyasviel/Fooocus/assets/155782314/54e79b2a-8fe7-4bdd-b6b7-9930a45907e4)
